utils/mdx-utils.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs')
+const path = require('path')
+const matter = require('gray-matter')
+
+// POSTS_PATH is useful when you want to get the path to a specific file
+const POSTS_PATH = path.join(process.cwd(), 'posts')
```

Comment:
use the parent directory of `mdx-utils.js` file regardless of where the script is invoked from?

```suggestion
const POSTS_PATH = path.join(path.dirname(__dirname), 'posts')
```

Reply:
And hm at least locally `path.dirname(__dirname)` returns `blog/.next/server`, which breaks routing. I'm not too familiar with best practices here, I frankly just copied over `process.cwd()` from a [Next example](https://github.com/vercel/next.js/blob/5907e9d3945d6471438cc087e652ca244e3d3c8e/examples/with-mdx-remote/utils/mdxUtils.js#L5) 😅

Reply:
good catch. my python bias made me believe this behaved the same way and it turns out that using `path.join(path.dirname(__dirname), "src/posts")` was actually breaking my `next` builds. I reverted back to using `process.cwd()` and everything seems to be working fine.